src/SprykerEco/Zed/Optivo/Dependency/Facade/OptivoToMoneyFacadeBridge.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+namespace SprykerEco\Zed\Optivo\Dependency\Facade;
+
+use Generated\Shared\Transfer\MoneyTransfer;
+
+class OptivoToMoneyFacadeBridge implements OptivoToMoneyFacadeInterface
+{
+    /**
+     * @var \Spryker\Zed\Money\Business\MoneyFacadeInterface
+     */
+    protected $moneyFacade;
+
+    /**
+     * @param \Spryker\Zed\Money\Business\MoneyFacadeInterface $moneyFacade
+     */
+    public function __construct($moneyFacade)
+    {
+        $this->moneyFacade = $moneyFacade;
+    }
+
+    /**
+     * @param int $value
+     *
+     * @return \Generated\Shared\Transfer\MoneyTransfer
+     */
+    public function fromInteger(int $value): MoneyTransfer
+    {
+        return $this->moneyFacade->fromInteger($value);
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\MoneyTransfer $moneyTransfer
+     *
+     * @return string
+     */
+    public function formatWithSymbol(MoneyTransfer $moneyTransfer): string
```

Comment:
Please use exactly the same method typehinting like in MoneyFacadeInterface.

Reply:
and what is wrong here?